dist/assembly.xml

```diff
@@ -19,57 +19,75 @@
 -->
 
 <assembly>
-  <id>distribution</id>
+  <id>zip-distrib</id>
+  <formats>
```

Comment:
I don't think <formats> here is needed. I believe Peter's last PR removed this but somehow the rebase for this PR brought it back. The pom.xml's assembly plugin configuration defines zip and tar.gz already.